mnefun/_report.py

```diff
@@ -67,15 +67,12 @@ def gen_html_report(p, subjects, structurals, run_indices=None,
             #
             # Head coils
             #
-            section = 'HPI coil SNR'
-            if p.report_params.get('coil_snr', True) and p.movecomp:
+            section = 'Good HPI count'
+            if p.report_params.get('good_hpi_count', True) and p.movecomp:
                 t0 = time.time()
+                _, _, fit_data = _head_pos_annot(p, fname, prefix='      ')
```

Comment:
> Traceback (most recent call last):
>   File "/home/ktavabi/Projects/bad-baby/mnefun_/process_mismatch.py", line 163, in <module>
>     print_status=True
>   File "/home/ktavabi/Projects/mnefun/mnefun/_mnefun.py", line 629, in do_processing
>     outs[ii] = func(p, subjects, structurals, run_indices)
>   File "/home/ktavabi/Projects/mnefun/mnefun/_report.py", line 73, in gen_html_report
>     _, _, fit_data = _head_pos_annot(p, fname, prefix='      ')
> NameError: name 'fname' is not defined

Reply:
@larsoner I am getting around this using `raw._filenames[0]` but IDK if this is a good situation for raw files with splits unless `_head_pos_annot` can deal with them downstream?